tools/theme/core.ts

```diff
@@ -14,4 +14,7 @@ import { Theme } from './src/Theme.js';
 
 import coreStyles from './src/theme.css.js';
 
+Theme.registerThemeFragment('spectrum', 'system', coreStyles);
+
+// `theme` is deprecated in favor of `system` but maintaining `theme` as a deprecated path.
```

Comment:
/**
@deprecation: `theme` is deprecated in favor of `system` but maintaining `theme` as a deprecated path.
**/